be/src/runtime/routine_load/data_consumer_group.cpp

```diff
@@ -239,22 +239,25 @@ Status KafkaDataConsumerGroup::start_all(StreamLoadContext* ctx) {
             } else {
                 Status st = Status::OK();
                 if (ctx->format == TFileFormatType::FORMAT_AVRO) {
```

Comment:
```suggestion
                if (ctx->format == TFileFormatType::FORMAT_AVRO && msg->len() > 0) {
```

Reply:
> 

To do so would enter else logic, which is not right
![image](https://user-images.githubusercontent.com/45813655/225536327-64a2572d-cb87-4cd3-a643-db6820457748.png)